crates/ruff_linter/src/rules/pycodestyle/rules/too_many_newlines_at_end_of_file.rs

```diff
@@ -1,6 +1,9 @@
+use std::iter::Peekable;
+
 use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
 use ruff_macros::{derive_message_formats, ViolationMetadata};
-use ruff_python_parser::{TokenKind, Tokens};
+use ruff_notebook::CellOffsets;
+use ruff_python_parser::{Token, TokenKind, Tokens};
 use ruff_text_size::{Ranged, TextRange, TextSize};
 
 /// ## What it does
```

Comment:
We should extend the rule documentation to mention its behavior in a notebook context